<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDashboard</class>
 <widget class="QDialog" name="UserDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QLabel" name="logo_searchScreen">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>110</y>
     <width>479</width>
     <height>109</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(C:/LibManager/LibManager/LBResources/LibManager_searchScreen.png);
background-color:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchLineEdit">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>250</y>
     <width>407</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.484, y1:0.534091, x2:0.479, y2:1, stop:0 rgba(90, 191, 47, 255), stop:1 rgba(13, 134, 0, 255));
font-size: 16px;
padding-left: 10px; 
padding-bottom: 2px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="navBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.484, y1:0.534091, x2:0.479, y2:1, stop:0 rgba(90, 191, 47, 255), stop:1 rgba(13, 134, 0, 255));</string>
   </property>
   <widget class="QLabel" name="LibManager_logo_navBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>12</y>
      <width>130</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(C:/LibManager/LibManager/LBResources/LibManager_navBar.png);
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="showBooksButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
    padding: 5px;
	border-color: transparent;
	border-radius: 0px;
}
QPushButton:hover {
	background-color: rgb(136, 202, 129);
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>All Books</string>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="showBorrowedBooksButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: transparent;
    color: white;
    padding: 5px;
	border-color: transparent;
	border-radius: 0px;
}
QPushButton:hover {
	background-color: rgb(136, 202, 129);
	border-radius: 0px;
}</string>
    </property>
    <property name="text">
     <string>Borrowed Books</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="allBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>576</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(231, 231, 231);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="bookListWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>191</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.484, y1:0.534091, x2:0.479, y2:1, stop:0 rgba(90, 191, 47, 255), stop:1 rgba(13, 134, 0, 255));</string>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="filterComboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>111</width>
     <height>44</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
    background-color:white;
    color: green;
    padding: 5px;
	border-color: transparent;
	border-radius: 0px;

</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="resultsTableWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>294</y>
     <width>521</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.484, y1:0.534091, x2:0.479, y2:1, stop:0 rgba(90, 191, 47, 255), stop:1 rgba(13, 134, 0, 255));</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>250</y>
     <width>111</width>
     <height>44</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color:white;
    color: green;
    padding: 5px;
	border-color: transparent;
	border-radius: 0px;
}
</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="barrowedbooks">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>50</y>
     <width>491</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.484, y1:0.534091, x2:0.479, y2:1, stop:0 rgba(90, 191, 47, 255), stop:1 rgba(13, 134, 0, 255));
color: white;</string>
   </property>
  </widget>
  <zorder>allBackground</zorder>
  <zorder>logo_searchScreen</zorder>
  <zorder>searchLineEdit</zorder>
  <zorder>navBar</zorder>
  <zorder>resultsTableWidget</zorder>
  <zorder>findButton</zorder>
  <zorder>filterComboBox</zorder>
  <zorder>bookListWidget</zorder>
  <zorder>barrowedbooks</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
